src/styles/typography.scss

```diff
@@ -76,6 +76,13 @@
   line-height: 24px;
 }
 
+%face-sans-14-bold {
+  font-family: $font-sans;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 24px;
+}
+
```

Comment:
Will this be used somewhere else, if not we may just define `font-weight` here.

Reply:
Yes, in auctions, though with different `line-height`. 

There's still no clearance on #1049 but I guess it would be done like here: https://github.com/aeternity/superhero-wallet/pull/1124/files#diff-72b5bff734d57e7b2cf89d4e386f24a599c0a029d5a53c30422bf427c04f385dR79-R81

I also spotted `%face-sans-15-bold` and `%face-sans-20-bold` are not used anywhere and few others have only 1 occurrence. Should it be a concern at the moment, given the new design implementation is underway?